app/services/platforms/facebook.ts

```diff
@@ -261,29 +263,23 @@ export class FacebookService extends StatefulService<IFacebookServiceState>
     this.SET_STREAM_PROPERTIES(title, description, game);
     await this.postPage(facebookPageId);
     if (this.state.liveVideoId && game) {
-      const headers = this.getHeaders(this.state.activePage.access_token);
-      const data = { title, description, game_specs: { name: game } };
-      const request = new Request(`${this.apiBase}/${this.state.liveVideoId}`, {
-        headers,
-        method: 'POST',
-        body: JSON.stringify(data),
-      });
-      return fetch(request)
-        .then(handlePlatformResponse)
-        .then(() => true);
+      return platformRequest(
+        {
+          url: `${this.apiBase}/${this.state.liveVideoId}`,
+          method: 'POST',
+          body: JSON.stringify({ title, description, game_specs: { name: game } }),
+        },
+        this.state.activePage.access_token,
+      ).then(() => true);
```

Comment:
we don't seem to be attaching headers to this request

Reply:
Headers will be attached in the `platformRequest` method